tests/iteminfo_test.cpp

```diff
@@ -69,33 +112,142 @@ TEST_CASE( "gun_damage_multiplier_not_integer", "[item][iteminfo]" )
         "Damage: <color_c_yellow>18</color>*<color_c_yellow>1.25</color> = <color_c_yellow>22</color>\n" );
 }
 
-TEST_CASE( "nutrients_in_regular_item", "[item][iteminfo]" )
-{
+TEST_CASE( "nutrients in food", "[item][iteminfo]" )
```

Comment:
Looks like you lost a bracket here
```suggestion
TEST_CASE( "nutrients in food", "[item][iteminfo]" )
{
```

Reply:
Good catch, thanks! Fixed in the latest commit.